src/aleph/web/controllers/ipfs.py

```diff
@@ -29,7 +30,23 @@ async def ipfs_add_file(request: web.Request):
     except KeyError:
         raise web.HTTPUnprocessableEntity(reason="Missing 'file' in multipart form.")
 
-    ipfs_add_response = await ipfs_service.add_file(file_field_to_io(file_field))
```

Comment:
`field_field_to_io` doesn't seem to be used anymore, do we want that? If so, should we remove it entirely?